src/site/stages/build/drupal/health-care-region.js

```diff
@@ -108,11 +108,6 @@ function createHealthCareRegionListPages(page, drupalPagePath, files) {
   statusPage.regionOrOffice = page.title;
   statusPage.entityUrl = generateBreadCrumbs(statusPath);
 
-  files[`${drupalPagePath}/status/index.html`] = createFileObj(
-    statusPage,
-    'health_care_facility_status.drupal.liquid',
-  );
-
```

Comment:
Can we remove the whole statusPage variable?

Reply:
It looks like it is being used to generate the other status page in addition to the old one, so I left it in place. I can poke around and see if I can remove more!

Reply:
Actually I think we could remove from [line 92](https://github.com/department-of-veterans-affairs/vets-website/pull/15544/files#diff-9ea2c3ce2010c72097f03bd4f4cf5d477f13f2a1f82a8bb63467457238403571R92) down to [line 110](https://github.com/department-of-veterans-affairs/vets-website/pull/15544/files#diff-9ea2c3ce2010c72097f03bd4f4cf5d477f13f2a1f82a8bb63467457238403571R110) 🙂

Reply:
Yeah I _think_ the other status page is generated via the direct result of a GraphQL query instead of being generated like this one here. So I think this one is deprecated completely now

Reply:
I will remove it and double check and then push it up!